k8s-tester/csi-ebs/tester.go

```diff
@@ -140,70 +183,26 @@ func (ts *tester) runPrompt(action string) (ok bool) {
 	return true
 }
 
-// https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/master/aws-ebs-csi-driver/values.yaml
-func (ts *tester) installChart() error {
-	values := map[string]interface{}{
-		"enableVolumeScheduling": true,
-		"enableVolumeResizing":   true,
-		"enableVolumeSnapshot":   true,
-	}
+var dirOrCreate = v1.HostPathDirectoryOrCreate
 
+func (ts *tester) installEBSHelmChart() error {
 	getAllArgs := []string{
 		ts.cfg.Client.Config().KubectlPath,
 		"--kubeconfig=" + ts.cfg.Client.Config().KubeconfigPath,
 		"--namespace=kube-system",
 		"get",
-		"all",
-	}
-	getAllCmd := strings.Join(getAllArgs, " ")
-
-	descArgsDs := []string{
```

Comment:
Can we keep these logging? I found them helpful when debugging some random driver issues :)

Reply:
Yea I was getting overload issues causing the goroutine to fail sometimes when I had too much stuff coming out.

Reply:
```
{"level":"info","ts":"2021-06-15T17:38:08.731-0500","caller":"csi-ebs/tester.go:210","msg":"[install] Deployment is not ready: kube-system/ebs-csi-controller. 0 out of 2 expected pods are ready"}
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x40f4b77]
```

Reply:
Not sure if this is related to my computer or just too much coming out of the commands.

Reply:
Put them back in for now

Reply:
Hmm, can we get the full stack trace? Let me merge and let's try again.